kitsune/customercare/models.py

```diff
@@ -1,145 +1,80 @@
-    banned = models.BooleanField(default=False, db_index=True)
-    ignored = models.BooleanField(default=False, db_index=True)
-
-    class Meta:
-        permissions = (
-            ("ban_account", "Can ban twitter accounts"),
-            ("ignore_account", "Can tag accounts to ignore"),
-        )
-
-
-class Tweet(ModelBase):
-    """An entry on twitter."""
-
-    tweet_id = models.BigIntegerField(primary_key=True)
-    raw_json = models.TextField()
-    # This is different from our usual locale, so not using LocaleField.
-    locale = models.CharField(max_length=20, db_index=True)
-    created = models.DateTimeField(default=datetime.now, db_index=True)
-    reply_to = models.ForeignKey(
-        "self", on_delete=models.CASCADE, null=True, related_name="replies"
-    )
-    hidden = models.BooleanField(default=False, db_index=True)
-
-    class Meta:
-        ordering = ("-tweet_id",)
-
-    @classmethod
-    def latest(cls):
-        """Return the most recent tweet.
-
-        Raise Tweet.DoesNotExist if there are None.
-
-        This is like Tweet.objects.latest(), except it sorts by tweet_id rather
-        than a date column.
-
-        """
-        return cls.objects.order_by("-tweet_id")[0:1].get()
-
-    def __str__(self):
-        tweet = json.loads(self.raw_json)
-        return tweet["text"]
-
-
-class Reply(ModelBase, SearchMixin):
-    """A reply from an AoA contributor.
-
-    The Tweet table gets truncated regularly so we can't use it for metrics.
-    This model is to keep track of contributor counts and such.
-    """
-
-    user = models.ForeignKey(
-        User, on_delete=models.CASCADE, null=True, blank=True, related_name="tweet_replies"
-    )
-    twitter_username = models.CharField(max_length=20)
-    tweet_id = models.BigIntegerField()
-    raw_json = models.TextField()
-    locale = models.CharField(max_length=20)
-    created = models.DateTimeField(default=datetime.now, db_index=True)
-    reply_to_tweet_id = models.BigIntegerField()
-
-    def __str__(self):
-        tweet = json.loads(self.raw_json)
-        return "@{u}: {t}".format(u=self.twitter_username, t=tweet["text"])
-
-    @classmethod
-    def get_mapping_type(cls):
-        return ReplyMetricsMappingType
-
-
-@register_mapping_type
-class ReplyMetricsMappingType(SearchMappingType):
-    @classmethod
-    def get_model(cls):
-        return Reply
-
-    @classmethod
-    def get_index_group(cls):
-        return "metrics"
-
-    @classmethod
-    def get_mapping(cls):
-        return {
-            "properties": {
-                "id": {"type": "long"},
-                "model": {"type": "string", "index": "not_analyzed"},
-                "url": {"type": "string", "index": "not_analyzed"},
-                "indexed_on": {"type": "integer"},
-                "created": {"type": "date"},
-                "locale": {"type": "string", "index": "not_analyzed"},
-                "creator_id": {"type": "long"},
-            }
-        }
+# import json
```

Comment:
Yes, this will produce a migration that will drop the tables. I could delete them but since we need one more step after the release to remove the app from the `INSTALLED_APPS`, I figured that we can remove the tables after that step in case we need them